packages/salesforcedx-vscode-apex-debugger/test/telemetry/telemetry.test.ts

```diff
@@ -6,7 +6,7 @@
  */
 import { assert, match, SinonStub, stub } from 'sinon';
 import TelemetryReporter from 'vscode-extension-telemetry';
-import { TelemetryService } from '../../../src/telemetry/telemetry';
+import { TelemetryService } from '../../src/telemetry/telemetry';
```

Comment:
We do not have any strict unit tests in any of our packages with names starting with "salesforcedx-vscode-*" (in other words, our extension packages), so I have moved all tests under the `unit` test subdirectory into the top level `test` directory and removed the `test/unit` directory.